packages/calcite-components/src/components/split-button/split-button.stories.ts

```diff
@@ -168,6 +176,31 @@ export const iconStartAndIconEnd = (): string => html`
   </div>
 `;
 
+export const placementTopStart = (): string => html`
```

Comment:
Do you think we could replace this and similar tests by enhancing the [`floatingUI` common test helper](https://github.com/Esri/calcite-design-system/blob/main/packages/calcite-components/src/tests/commonTests/floatingUI.ts? If so, we can create a separate issue for this.

Reply:
yeah that makes sense as a separate issue. Can you create one?